physionet-django/project/views.py

```diff
@@ -1772,6 +1772,9 @@ def published_project(request, project_slug, version, subdir=''):
     """
     Displays a published project
     """
+    from project.cloud.s3 import get_bucket_name_and_prefix, check_s3_bucket_with_prefix_exists, has_aws_credentials
```

Comment:
imports don't belong here.

Reply:
@tompollard I've moved import to the top of the module.